build-tools/packages/version-tools/src/bumpTypes.ts

```diff
@@ -5,6 +5,11 @@
 
 import type { SemVer } from "semver";
 
+/**
+ * A type alias for strings that represent package versions.
```

Comment:
Nit: any documentation regarding the form the contents of these strings are expected to take that we can link to from here?